codelabs/Module1_SauceConnect.md

```diff
@@ -16,13 +16,19 @@ Duration: 0:01:00
 
 This course covers the basic setup and use of the Sauce Connect Proxy Tunnel, which is a tool created by Sauce Labs that customers can use to establish a secure, private connection between your app and the Sauce Labs Cloud.
 
-This tutorial gives examples written in Java, using the JUnit4 test runner, as well as the Maven build tool. If you would like to follow along, you can [download or fork and clone this project](https://github.com/walkerlj0/Selenium_Course_Example_Code)
+### What You'll Need
+In order to follow along with the course, you will need a few things set up ahead of time:
 
-* Look in the _/java/Mod4/4.06_ directory, and use the version of the test suite **in 4.06** to work from.
-* If you don't have Java and Maven set up on your machine to run test code, [follow this lesson](https://training.saucelabs.com/codelabs/Module1-SeleniumJava/index.html?index=..%2F..SeleniumJava#4)
+* A Sauce [Username and Access Key](https://app.saucelabs.com/user-settings)
+* The [Tunnel Name](https://app.saucelabs.com/tunnels) of a running tunnel
+* A copy of [Sauce Connect](https://docs.saucelabs.com/secure-connections/sauce-connect/installation/)
+* Automated tests. (See the example [Selenium Java test code](https://github.com/walkerlj0/sauceconnect-github-actions))
+* A computer with unrestricted access to saucelabs.com
```

Comment:
Is this true? Sounds like it would negate the purpose of SC? Perhaps you meant the appropriate SC domains allow list?

Reply:
Good call, will say 'access to Sauce Labs' and link to Allow Listing Doc